src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -58,14 +58,30 @@ export default function TypeOfCarePage() {
         (careA, careB) =>
           careA.name.toLowerCase() > careB.name.toLowerCase() ? 1 : -1,
       );
+
+      const covidLabel = (
+        <>
+          COVID-19 vaccine
+          <span className="usa-label vads-u-margin-left--1">New</span>
+        </>
+      );
+
       return {
         type: 'object',
         required: ['typeOfCareId'],
         properties: {
           typeOfCareId: {
             type: 'string',
             enum: sortedCare.map(care => care.id || care.ccId),
-            enumNames: sortedCare.map(care => care.label || care.name),
+            enumNames: sortedCare.map(care => {
+              if (
+                care.label === 'COVID-19 vaccine' ||
```

Comment:
I think you could do `care.id === 'covid'`, instead of checking these label names.